speechmatics/cli.py

```diff
@@ -335,6 +336,11 @@ def get_transcription_config(
             topic_detection_config.topics = topics
         config["topic_detection_config"] = topic_detection_config
 
+    auto_chapters_config = config.get("auto_chapters_config", {})
+    args_auto_chapters = args.get("detect_chapters")
+    if args_auto_chapters or auto_chapters_config:
```

Comment:
There is a situation where this could make a problem in the future: if we add an empty `auto_chapters_config` to the config.json, we don't enter this if, since  an empty dictionary evaluates to false `bool({}) == False`. I think this would be clearer the following way
```suggestion
    auto_chapters_config = config.get("auto_chapters_config", None)
    args_auto_chapters = args.get("detect_chapters")
    if args_auto_chapters or auto_chapters_config is not None:
```

Reply:
thanks, good shout. I've fixed it for all SU features